<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editStudentDialog</class>
 <widget class="QDialog" name="editStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>709</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>URW Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Student Data</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(41, 48, 47);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 255, 198);</string>
             </property>
             <property name="text">
              <string>Roll No.:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rollNumSB">
             <property name="font">
              <font>
               <family>URW Gothic</family>
               <pointsize>25</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 255, 198);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>58</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="okBtn">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 255, 198);
color: rgb(0, 73, 66);</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>128</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 73, 66);
background-color: rgb(0, 255, 198);</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>URW Gothic</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 198);</string>
       </property>
       <property name="title">
        <string>Data:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <family>URW Gothic</family>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Basic:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nameLE">
               <property name="font">
                <font>
                 <family>URW Gothic</family>
                 <pointsize>25</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(41, 48, 47);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Class:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="classCB">
               <property name="font">
                <font>
                 <family>URW Gothic</family>
                 <pointsize>25</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>IT</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COMP</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>MECH</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ENTC</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>CIVIL</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>458</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Div.:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="divCB">
               <property name="font">
                <font>
                 <family>URW Gothic</family>
                 <pointsize>25</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>A</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>B</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>C</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>D</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>E</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>F</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>G</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>H</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>I</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>J</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>518</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Marks:</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>41</y>
             <width>695</width>
             <height>53</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Physics:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pSB">
              <property name="font">
               <font>
                <family>URW Gothic</family>
                <pointsize>25</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>518</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>111</y>
             <width>701</width>
             <height>53</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Chemistry:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="cSB">
              <property name="font">
               <font>
                <family>URW Gothic</family>
                <pointsize>25</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>488</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>11</x>
             <y>181</y>
             <width>701</width>
             <height>53</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Marks:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mSB">
              <property name="font">
               <font>
                <family>URW Gothic</family>
                <pointsize>25</pointsize>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>538</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
